hack/update-submodule.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# Update submodules HEAD to remote branches
+
+function help() {
+    echo "Usage: update_submodules.sh [OPTIONS]... branch_name..."
+    echo "Update submodules HEAD to remote branches"
+    echo "Must be ran in repo root directory"
+    echo ""
+    echo "Options:"
+    echo "-h   Shows usage text"
+    echo "Example:"
+    echo "update_submodules.sh master release-4.6"
+}
+
+function update_branch_submodules() {
+    submodules=(kubernetes ovn-kubernetes windows-machine-config-bootstrapper windows_exporter containernetworking-plugins)
```

Comment:
Can't you figure this out? Otherwise we would have to keep updating this script.
`git config --file .gitmodules --get-regexp path | awk '{ print $2 }'`